install_config/topics/configuring_for_vsphere_registry.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * install_config/configuring_for_vsphere.adoc
+
+[id='vsphere-configuring-registry_{context}']
+= Configuring the {product-title} registry for vSphere
+The steps below define the steps define the manual process to 
+create storage to be used by the registry if a storage class is unavailable 
+or not used.
+
+[source, bash]
+----
+# VMFS
+cd /vmfs/volumes/datastore1/
+mkdir kubevols # Not needed but good hygiene
+
+# VSAN
+cd /vmfs/volumes/vsanDatastore/
+/usr/lib/vmware/osfs/bin/osfs-mkdir kubevols # Needed
+
+cd kubevols
+
+vmkfstools -c 25G registry.vmdk
+----
+
+== Configuring the {product-title} registry for vSphere using Ansible
+
+.Procedure
+
+To configure the Ansible inventory for the registry to use a vSphere volume:
+
+----
+[OSEv3:vars]
+# vSphere Provider Configuration
+openshift_hosted_registry_storage_kind=vsphere <1>
+openshift_hosted_registry_storage_access_modes=['ReadWriteOnce'] <2>
+openshift_hosted_registry_storage_annotations=['volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/vsphere-volume'] <3>
+openshift_hosted_registry_replicas=1 <4>
+----
+<1> The storage type.
+<2> vSphere volumes only support `RWO`.
+<4> The annotation for the volume.
+<4> The number of replicas to configure.
+
+NOTE: The brackets in the configuration file above are required.
```

Comment:
Note boxes are formatted as:

    [NOTE]
    =====
    The brackets in the configuration file above are required.
    ====

See https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions